notebooks/supernova_model_sims.ipynb

```diff
@@ -9,72 +9,298 @@
    "source": [
```

Comment:
<p>is it perhaps possible that this notebook has examples for LSST and Roman at the same time as more people are perhaps interested in simulating lensed SNe for LSST than Roman?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LSST-strong-lensing/slsim/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Yes, I can do that, but the notebook will become quite large as it will include lots of examples

Reply:
to make a bit fewer plots, perhaps, you can also over-plot different bands or models on top of each other - thank you!